frontend/src/bundles/users/pages/profile.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button } from '~/bundles/common/components/components';
+
+import styles from './style.module.scss';
+
+type User = {
+    firstName?: string;
+    lastName?: string;
+    email?: string;
+};
+
+const handleFormSubmit = (payload: User): void => {
+    // Handle form submission logic here
+};
+
+const Profile: React.FC = () => {
+    return (
+        <div className={styles.profile}>
+            <div className={styles.profile__container}>
+                <div className={styles.profile__title}>
+                    <h2>Account settings</h2>
```

Comment:
Should be moved to a separate reusable component, for example PageTitle

Reply:
done